core/local/analysis.js

```diff
@@ -170,12 +120,82 @@ function analyseEvents (events /*: LocalEvent[] */, pendingChanges /*: LocalChan
   }
 
   log.trace('Flatten changes map...')
-  for (let a of changesByInode.values()) changes.push(a)
+  const changes /*: LocalChange[] */ = changesFound.flush()
 
   stopMeasure()
   return changes
```

Comment:
Why no `stopMeasure()` first, then `return changesFound.flush();`?

Reply:
Because we want it to be part of the measurement?